swiftide-indexing/src/pipeline.rs

```diff
@@ -207,7 +212,6 @@ impl Pipeline {
     ///
```

Comment:
A hint here that it will prefer the batch size from the transformer if it's present could save some headaches.